{"name": "test_login_with_valid_email_and_password", "status": "broken", "statusDetails": {"message": "AttributeError: 'SubRequest' object has no attribute 'mode'. Did you mean: 'node'?", "trace": "self = <allure_commons._allure.fixture object at 0x0000022CD4323F00>, args = (), kwargs = {}\n\n    def __call__(self, *args, **kwargs):\n        self.parameters = func_parameters(self._fixture_function, *args, **kwargs)\n    \n        with self:\n>           return self._fixture_function(*args, **kwargs)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n.venv\\Lib\\site-packages\\allure_commons\\_allure.py:231: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nrequest = <SubRequest 'log_on_failure' for <Function test_login_with_valid_email_and_password>>\n\n    @pytest.fixture()\n    def log_on_failure(request):\n        yield\n>       item=request.mode\n             ^^^^^^^^^^^^\nE       AttributeError: 'SubRequest' object has no attribute 'mode'. Did you mean: 'node'?\n\ntests\\conftest.py:11: AttributeError"}, "start": 1758948641017, "stop": 1758948644353, "uuid": "3114e273-f0c3-49c8-aa57-0a7f4b2f4a6b", "historyId": "27914f0967bc11c271167b994e8fde90", "testCaseId": "27914f0967bc11c271167b994e8fde90", "fullName": "tests.test_login.TestLogin#test_login_with_valid_email_and_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-PQJ2IUR"}, {"name": "thread", "value": "12224-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py", "TestLogin"]}